pom.xml

```diff
@@ -131,21 +145,21 @@
     <aspectj.version>1.9.6</aspectj.version>
     <raml-module-builder.version>33.0.2</raml-module-builder.version>
     <vertx.version>4.1.0</vertx.version>
-    <vertx-folio.version>3.9.4-FOLIO</vertx-folio.version>
+    <vertx-folio.version>4.1.0-FOLIO</vertx-folio.version>
```

Comment:
This is no longer in use and should be removed.

Reply:
Good, I removed this property.